runtime/lua/vim/lsp/protocol.lua

```diff
@@ -688,7 +688,9 @@ function protocol.make_client_capabilities()
         };
         hierarchicalDocumentSymbolSupport = true;
       };
-      workspaceSymbol = {
+    };
+    workspace = {
+      Symbol = {
```

Comment:
looking at the comment and spec, it's a small case `s`

Reply:
right, I always forget these are case sensitive :/